includes/widgets/class-wc-widget-price-filter.php

```diff
@@ -152,11 +152,7 @@ protected function get_filtered_price() {
 		$meta_query = isset( $args['meta_query'] ) ? $args['meta_query'] : array();
 
 		if ( ! is_post_type_archive( 'product' ) && ! empty( $args['taxonomy'] ) && ! empty( $args['term'] ) ) {
-			$tax_query[] = array(
-				'taxonomy' => $args['taxonomy'],
-				'terms'    => array( $args['term'] ),
-				'field'    => 'slug',
-			);
+			$tax_query[] = WC()->query->get_main_query()->tax_query->queries;
```

Comment:
Still need to test, but you can replace it with:

```php
$tax_query[] = WC()->query->get_main_tax_query();
```

Reply:
Thank you for the suggestion. I will replace and test from my side.